### DISCUSSION TITLE
does symja have a vector magnitude function?

### DISCUSSION THREAD

**Discussion by xwaeaewcrhomesysplug** [axkr/symja_android_library#287] [Q&A] [answered]
Like given vector eg a={1,2}

length=(1^2+2^2)^0.5
That vector magnitude.is there such a function i expect it to exist maybe in some core? because many other vector function depends on it.

**Comment by axkr** [comment, marked as answer]
Do you mean the Norm function?

- https://github.com/axkr/symja_android_library/blob/master/symja_android_library/doc/functions/Norm.md

```mma
Norm({a,b,c})
```

**Comment by xwaeaewcrhomesysplug** [comment]
yes exactly,it seems like i am not knowledgable enough with the mathematical terms.
Thanks.
